deploy/examples/create-external-cluster-resources.py

```diff
@@ -894,6 +894,10 @@ def get_rbd_node_caps_and_entity(self):
             rbd_pool_name = self._arg_parser.rbd_data_pool_name
             alias_rbd_pool_name = self._arg_parser.alias_rbd_data_pool_name
             cluster_name = self._arg_parser.cluster_name
+            if rbd_pool_name == "" or cluster_name == "":
```

Comment:
How about a separate check for these two args so we can have separate error messages for each one? As a combined error message, it can be confusing if you already set one of them.